Documentation/clustering.md

```diff
@@ -43,46 +43,46 @@ etcd listens on [`listen-client-urls`](configuration.md#-listen-client-urls) to
 On each machine you would start etcd with these flags:
 
 ```
-$ etcd -name infra0 -initial-advertise-peer-urls http://10.0.1.10:2380 \
-  -listen-peer-urls http://10.0.1.10:2380 \
-  -listen-client-urls http://10.0.1.10:2379,http://127.0.0.1:2379 \
-  -advertise-client-urls http://10.0.1.10:2379 \
-  -initial-cluster-token etcd-cluster-1 \
-  -initial-cluster infra0=http://10.0.1.10:2380,infra1=http://10.0.1.11:2380,infra2=http://10.0.1.12:2380 \
-  -initial-cluster-state new
+$ etcd2 --name infra0 --initial-advertise-peer-urls http://10.0.1.10:2380 \
```

Comment:
Ah, maybe I see what you mean now. It's super confusing anyhow considering that CoreOS docs links to this documentation, and hence the reference commands don't work.